diff --git a/android/CMakeLists.txt b/android/CMakeLists.txt
index 9b7794cbf6f42029aef19922cbb879ebb620f4b9..945c10cbb5c5c3feae62411bc6b67bd0dfba7c89 100644
--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.9.0)
 project(ReactNativeMmkv)
 
 set(CMAKE_VERBOSE_MAKEFILE ON)
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)
 
 # Compile sources
 add_library(
